llvm/lib/Passes/PassBuilderPipelines.cpp

```diff
@@ -217,6 +218,10 @@ static cl::opt<bool> EnableLoopFlatten("enable-loop-flatten", cl::init(false),
                                        cl::Hidden,
                                        cl::desc("Enable the LoopFlatten Pass"));
 
+static cl::opt<bool> EnableLoopMetadata(
+    "enable-loop-metadata", cl::Hidden, cl::init(false),
```

Comment:
Is there a reason to have this off? I expect this as a default on feature :-)

Reply:
no, I do not have regressions and only improvements.